tox.ini

```diff
@@ -83,9 +83,14 @@ envlist =
     pypy3-test-instrumentation-fastapi
 
     ; opentelemetry-instrumentation-flask
-    py3{8,9,10,11}-test-instrumentation-flask-{213,220}
-    py3{8,9,10,11}-test-instrumentation-flask-{300}
-    pypy3-test-instrumentation-flask-{213,220}
+    ; The numbers at the end of the environment names
+    ; below mean these dependencies are being used:
+    ; 0: Flask ==2.1.3 Werkzeug <3.0.0
+    ; 1: Flask ==2.2.0 Werkzeug <3.0.0
+    ; 2: Flask >=3.0.0 Werkzeug >=3.0.0
```

Comment:
Would still prefer to also name them flask-21 flask-22 flask-33 but approved

Reply:
Thanks, we can check this approach again later :v: